Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -44,6 +44,14 @@
         "Icon": "/SamplePages/HeaderedTextBlock/HeaderedTextBlock.png"
       },
       {
+        "Name": "MasterDetailsView",
```

Comment:
I'm out of ideas now - what can I do more? Seems that I've now copy-pasted the file from dev branch, then added those few lines responsible for SystemInformation.